service/matching/taskListManager.go

```diff
@@ -725,3 +739,13 @@ func createServiceBusyError(msg string) *s.ServiceBusyError {
 func (c *taskListManagerImpl) isTaskAddedRecently(lastAddTime time.Time) bool {
 	return time.Now().Sub(lastAddTime) <= c.config.MaxTasklistIdleTime()
 }
+
+func domainTaggedMetricScope(cache cache.DomainCache, domainID string, client metrics.Client, scope int) metrics.Scope {
+	entry, err := cache.GetDomainByID(domainID)
+	switch {
```

Comment:
interesting use of switch. any reason to do this against a simple `if err != nil`

Reply:
probably not. I was thinking I will have more cases when I initially started writing this, then forgot to cleanup. I will change this to simple err != nil check